README.md

```diff
@@ -0,0 +1,15 @@
+# Fantasy Fitness
+
+## Development
+
+For development run the client and server on two different applications.
+
+To run the client application in development, cd into the client folder and `npm start`. 
+
+To run the server application in development, cd into the server folder and `npm start:dev`. The server will track file changes are restart the app. If no PORT environment variable is specified, then it will serve on port 5001.  For DB connections, the database used is the DATABASE_URL environment variable. If not defined, it will use the URI specified in `./config/index.json` under property `dbUri`.
```

Comment:
npm start:dev doesn't seem to work. The terminal is saying it cannot find the command.

Reply:
should be `npm run start:dev`